plugins/module_utils/crypto/support.py

```diff
@@ -223,6 +223,18 @@ def parse_name_field(input_dict):
     return result
 
 
+def parse_openssh_version(version_string):
```

Comment:
Could you put that into a new file `plugins/module_utils/openssh.py`?

Reply:
Done.